main/src/hadd.cxx

```diff
@@ -1,39 +1,64 @@
 /**
   \file hadd.cxx
-  \brief This program will add histograms (see note) and Trees from a list of root files and write them to a target root file.
-  The target file is newly created and must not be
-  identical to one of the source files.
+  \brief This program will merge compatible ROOT objects, such as histograms and Trees,
+         from a list of root files and write them to a target root file.
+         In order for a ROOT object to be mergeable, it must implement the Merge() function.
+         In addition, some other non-TObjects are also mergeable (such as RNTuple).
```

Comment:
I think it good to call `RNTuple` but it is not "in addition" since it literally goes through the "Merge" function mechanism.

Reply:
Right, I forgot to delete that line when I corrected the part about TObject and Merge, thanks.